integration-tests/test-suite/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fluent-bit-test-suite",
```

Comment:
Rename appropriately. Note that we always refer to "Fluent Bit", but we probably shouldn't (we're testing the Infra-Agent embedded log forwarder, which happens to be Fluent Bit)